src/Pages/Login/Login.js

```diff
@@ -1,13 +1,136 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { SIGN_IN} from './Data/config';
+import './Login.scss';
 
 class Login extends Component {
-    render() {
-        return (
-            <div>
-                
-            </div>
-        );
+  constructor() {
+    super();
+    this.state = {
+      id:"",
+      password:"",
+      showPw: true
     }
+  }
+
+  CheckPassword = () => {
+    this.setState({showPw : !this.state.showPw});
+  }
+
+  LoginInput = e => {
+    const { id, value } = e.target;
+    
+    this.setState({ [id] : value})
+  }
+
+  LoginButton = () => {
+    const { id, password} = this.state;
+
+    fetch(SIGN_IN,{
+      method: "POST",
+      body: JSON.stringify({
+        email: id,
+        password: password,
+      })
+    })
+      .then(res => res.json())
+      .then(result => {
+        localStorage.setItem("token", result.token);
+        console.log(result);
+        if(result.error === "INVALID_EMAIL"){
+          alert("아이디가 잘못 입력 됬습니다.");
+          return;
+        }
+        if(result.error === "INVALID_PASSWORD"){
+          alert("비밀번호를 잘못 입력 하셨습니다.");
+          return;
+        }
+        if(result.message === "SUCCESS") {
+          alert("로그인 성공!");
+          this.props.history.push('/main');
+
+        }
+      })
```

Comment:
```suggestion
      .then(res => res.json())
      .then(result => {
        const errorMsg = {
          INVALID_EMAIL: "아이디가 잘못 입력 됐습니다.",
          SUCCESS: "비밀번호를 잘못 입력 하셨습니다."
        }
        
        if (result.error) return alert(errorMsg[result.error])
        
        if(result.message === "SUCCESS") {
          localStorage.setItem("token", result.token);
          alert("로그인 성공!");
          this.props.history.push('/main');
        }
      })
```
- console.log는 테스트 끝나면 꼭 지워주세요.
- 어제 말씀드렸던 객체를 조건문으로 쓰는 방법입니다. 참고해보시고 비슷한 패턴이 나오면 꼭 한번 사용해보세요.
- 로컬스토리지에 토큰을 담는 것은 로그인 성공했을 경우에 처리하는 것이 절차상 맞아보입니다.

Reply:
수정 완료 했습니다!

Reply:
수정 완료했습니다!